src/shipit/frontend/src/views/NewRelease/index.js

```diff
@@ -6,7 +6,7 @@ import {
 import { object } from 'prop-types';
 import { NavLink } from 'react-router-dom';
 
-import config from '../../config';
+import { SHIPIT_API_URL, TREEHERDER_URL, PRODUCTS } from '../../config';
```

Comment:
I don't think `TREEHERDER_URL` or `PRODUCTS` are top-level values in that module.
They are include via https://github.com/mozilla/release-services/blob/master/src/shipit/frontend/src/config.js#L16

Reply:
Doh. Let's use the simpler version then